### PR TITLE
Adds a deleteById to kuss.

### PR DESCRIPTION
**Pull request by mszymulanski**
- Adds the deleteById method.
  - Update the version of pimp-my-sql to 0.3.1

### CODE REVIEW

**Review by scull7 — APPROVED** [review]

**Review by jxbadam — APPROVED** [review]
:shipit: 🔥

File: lib/mysql/index.js [hunk 1]
```diff
@@ -297,6 +297,12 @@ const findById = R.curry((mysql, table, projection, id) =>
 )
 
 
+// Permenently delete a record from given table.
+const deleteById = R.curry((mysql, table, id) =>
+  Query.deleteById(mysql, table, id)
+)
```
Comment (jonajonlee) [inline lib/mysql/index.js:305]: this should throw a NotFound error like the other stores if the id does not exist.

**Review by jonajonlee — CHANGES_REQUESTED** [review]
unify errors of deleteById across the stores

**Review by jonajonlee — APPROVED** [review]
:shipit:

**Review by scull7 — APPROVED** [review]
:shipit:

**Pull request merged**